numba/tests/test_types.py

```diff
@@ -561,15 +561,6 @@ def check_error(sig, msg):
 
 
 class TestRecordDtype(unittest.TestCase):
-    def test_record_type_equiv(self):
-        rec_dt = np.dtype([('a', np.int32), ('b', np.float32)])
-        rec_ty = typeof(rec_dt)
-        art1 = rec_ty[::1]
-        arr = np.zeros(5, dtype=rec_dt)
-        art2 = typeof(arr)
-        self.assertEqual(art2.dtype.dtype, rec_ty)
```

Comment:
Changing to
```suggestion
        self.assertEqual(art2.dtype.dtype, rec_ty.dtype)
```
makes the test pass.